source/codegen/service_helpers.py

```diff
@@ -24,6 +24,27 @@ def create_args(parameters):
         result = f'{result}{parameter_name}, '
     return result[:-2]
 
+def create_args_without_first_parameter(parameters):
+    result = ''
```

Comment:
I think you can replace this body with something like
```
return create_args(parameters[1:])
```

Reply:
I updated caller to use `create_args(parameters[1:])`